lib/home/profile/info/heart_center_info.dart

```diff
@@ -1,14 +1,253 @@
 import 'package:flutter/material.dart';
-import 'package:thc/utils/widgets/fun_placeholder.dart';
+import 'package:url_launcher/url_launcher.dart';
+import 'package:http/http.dart' as http;
 
-class HeartCenterInfo extends StatelessWidget {
+class HeartCenterInfo extends StatefulWidget {
   const HeartCenterInfo({super.key});
+  
+  @override
+  _HeartCenterInfoState createState() => _HeartCenterInfoState();
+}
+
+class _HeartCenterInfoState extends State<HeartCenterInfo> {
+  final _formKey = GlobalKey<FormState>();
```

Comment:
I didn't know anything about `GlobalKey`s until a few months ago!

Fantastic job with this 👍